cli/pcluster/config/update_policy.py

```diff
@@ -120,10 +132,41 @@ def _check_min_count(change, patch):
     return new_min >= old_min and new_max - new_min >= old_max - old_min
 
 
+def _is_bucket_pcluster_generated(stack_name):
+    params = utils.get_stack(stack_name).get("Parameters")
+    return utils.get_cfn_param(params, "RemoveBucketOnDeletion") == "True"
+
+
+def _check_generated_bucket(change, patch):
+
+    # If bucket is generated (no cluster_resource_bucket specified when creating) and no change in config
+    # Old value is retrieve from CFN's ResourcesS3Bucket param, and new value is from config(not specified)
+    # Diff based on config should be:
+    #
+    # old value                         new value
+    # ---------------------------------------------
+    # {ResourcesS3Bucket}               -
+    #
+    # Actual ResourcesS3Bucket will not change when updating, so no actual change when update is applied
+    # Since there is no change, user should not be informed of a change/diff
+    # Print no diff and proceed with updating other parameters
+    if _is_bucket_pcluster_generated(patch.stack_name) and (not change.new_value or change.new_value == "NONE"):
```

Comment:
Why are we allowing NONE as value? --> we should prevent this value in the s3_bucket_validator